src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -367,15 +367,6 @@ public override Symbol AssociatedSymbol
             }
         }
 
-        public override bool AreLocalsZeroed
```

Comment:
Deleted redundant implementation that was inadvertently left around after a merge